File: app/src/main/java/com/dewari/ajay/androidnetworkcommunication/models/ToDo.java
package com.dewari.ajay.androidnetworkcommunication.models;

import com.google.gson.annotations.SerializedName;

public class ToDo {

    @SerializedName("id")
    private int id;

    @SerializedName("userId")
    private int userId;

    @SerializedName("title")
    private String title;

    @SerializedName("completed")
    private boolean completed;
}

File: app/src/main/java/com/dewari/ajay/androidnetworkcommunication/models/ToDo.kt
package com.dewari.ajay.androidnetworkcommunication.models

import com.google.gson.annotations.SerializedName

class ToDo {
    @SerializedName("id")
    val id = 0

    @SerializedName("userId")
    val userId = 0

    @SerializedName("title")
    val title: String? = null

    @SerializedName("completed")
    val isCompleted = false

}